unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/BagAddOccurrencesProcedureTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020 Goldman Sachs and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.block.procedure;
+
+import org.eclipse.collections.api.bag.MutableBag;
+import org.eclipse.collections.api.block.function.primitive.BooleanFunction;
+import org.eclipse.collections.api.factory.Bags;
+import org.eclipse.collections.api.list.primitive.ImmutableBooleanList;
```

Comment:
There are several unused imports that are failing the checkstyle build. Please review the build failure and optimize the imports in the class.